src/Sylius/Bundle/AdminBundle/templates/shared/crud/common/sidebar/menu.html.twig

```diff
@@ -1,4 +1,4 @@
 <div class="collapse navbar-collapse" id="sidebar-menu">
-    {% include '@SyliusAdmin/shared/crud/common/sidebar/search.html.twig' %}
+    {% hook 'search_menu' %}
```

Comment:
```suggestion
```
Then we don’t need to fire the hook here anymore. Moreover, it’s not coupled with an order anymore, so if someone wants to change it (kinda weird to have a search bar below the menu), it works just by adjusting the YAML config :)